build-and-push-image/action.yml

```diff
@@ -0,0 +1,107 @@
+name: "Build and push image to SetOps registry"
+description: "Builds the docker images and pushes it to the SetOps registry while creating a tag for each stage / app combination"
+
+inputs:
+  setops-organization:
+    description: The SetOps organization
+    required: true
+  setops-project:
+    description: The SetOps project name
+    required: true
+  setops-username:
+    description: The SetOps username, usually obtained via secrets.SETOPS_USER
+    required: true
+  setops-password:
+    description: The SetOps password, usually obtained via secrets.SETOPS_PASSWORD
```

Comment:
secrets.SETOPS_PASSWORD isn't known on this level, so I'd remove it from the description.